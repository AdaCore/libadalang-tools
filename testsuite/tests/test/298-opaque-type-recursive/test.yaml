description: |
  Check that TGen supports types that are derived from an opaque type.

default-gpr: true

driver: gnattest_tgen
