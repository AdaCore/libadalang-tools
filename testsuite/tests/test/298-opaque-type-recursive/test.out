pkg.ads:15:4: info: corresponding test PASSED
pkg.ads:15:4: info: corresponding test PASSED
pkg.ads:15:4: info: corresponding test PASSED
pkg.ads:15:4: info: corresponding test PASSED
pkg.ads:15:4: info: corresponding test PASSED
pkg.ads:16:4: info: corresponding test PASSED
pkg.ads:16:4: info: corresponding test PASSED
pkg.ads:16:4: info: corresponding test PASSED
pkg.ads:16:4: info: corresponding test PASSED
pkg.ads:16:4: info: corresponding test PASSED
pkg.ads:14:4: info: corresponding test PASSED
pkg.ads:14:4: info: corresponding test PASSED
pkg.ads:14:4: info: corresponding test PASSED
pkg.ads:14:4: info: corresponding test PASSED
pkg.ads:14:4: info: corresponding test PASSED
pkg.ads:19:4: info: corresponding test PASSED
pkg.ads:19:4: info: corresponding test PASSED
pkg.ads:19:4: info: corresponding test PASSED
pkg.ads:19:4: info: corresponding test PASSED
pkg.ads:19:4: info: corresponding test PASSED
pkg.ads:17:4: info: corresponding test PASSED
pkg.ads:17:4: info: corresponding test PASSED
pkg.ads:17:4: info: corresponding test PASSED
pkg.ads:17:4: info: corresponding test PASSED
pkg.ads:17:4: info: corresponding test PASSED
pkg.ads:18:4: info: corresponding test PASSED
pkg.ads:18:4: info: corresponding test PASSED
pkg.ads:18:4: info: corresponding test PASSED
pkg.ads:18:4: info: corresponding test PASSED
pkg.ads:18:4: info: corresponding test PASSED
pkg.ads:14:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:61)
pkg.ads:15:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:97)
pkg.ads:16:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:133)
pkg.ads:17:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:169)
pkg.ads:18:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:205)
pkg.ads:19:4: error: corresponding test FAILED: Test not implemented. (pkg-test_data-tests.adb:241)
36 tests run: 30 passed; 6 failed; 0 crashed.
Test runner dumped 30 tests
